test/Oscoin/Test/HTTP/Helpers.hs

```diff
@@ -171,46 +169,36 @@ liftNode na = Session $ ReaderT $ \h -> liftIO (Node.runNodeT h na)
 
 genDummyTx
     :: (IsCrypto c, MonadIO m)
-    => PropertyM m (Hashed c (API.RadTx c), API.RadTx c)
+    => PropertyM m (Hashed c (Tx c), Tx c)
 genDummyTx = do
-    radValue <- pick arbitrary
-    createValidTx radValue
+    txVal <- pick arbitrary
+    createValidTx txVal
 
 createValidTx
     :: forall c m.
        (IsCrypto c, MonadIO m)
-    => Rad.Value
-    -> m (Hashed c (API.RadTx c), API.RadTx c)
-createValidTx radValue = liftIO $ do
+    => TxPayload c (Tx c) -- this is not correct
```

Comment:
:eyes:

Reply:
it used not to be correct but with the advent of `TxPayload` the comment is indeed moot. oops!